cpp/mail_krimdok_newsletter.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+
+BACKUP_DIR=/usr/local/var/lib/tuelib/sent_newsletters
```

Comment:
a subdir inside the newsletters dir might be better

Reply:
Agreed.